### DISCUSSION TITLE
DSS-1.0.1&Linkis-1.0.3  登录首页异常，多个HTTP接口出现 404

### DISCUSSION THREAD

**Discussion by shenbinglife** [WeBankFinTech/DataSphereStudio#569] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/16119667/162714077-fcdc664a-2bef-4c31-ba5c-e241d380e016.png)
![image](https://user-images.githubusercontent.com/16119667/162714406-7655bb16-de8c-43be-928f-205bbde0a53c.png)

**Comment by shenbinglife** [comment, marked as answer]
这是project服务没有启动起来
![2d29606c1aacc1b42897222b71ae95c](https://user-images.githubusercontent.com/16119667/162865722-9f47ce33-a45f-47ee-a920-01997c80a861.png)

project服务日志报BML上传失败问题， 修复了BML hadoop目录并重启，确实有了project服务

**Reply by ynxu15** [reply]
请问是这个问题是如何修复？我也遇到了这个问题，一直启动不了project服务
